spec/src/main/doc/core/spi.asciidoc

```diff
@@ -1062,9 +1047,9 @@ If any +ProcessInjectionPoint+ method is called outside of the observer method i
 
 ==== +ProcessInjectionTarget+ event
 
-The container must fire an event for every Java EE component class supporting injection that may be instantiated by the container at runtime, including every managed bean declared using +@ManagedBean+, EJB session or message-driven bean, bean, interceptor or decorator.
+The container must fire an event for every component class supporting injection that may be instantiated by the container at runtime, including every managed bean declared using +@ManagedBean+, bean, interceptor or decorator.
```

Comment:
Out of scope for this ticket. But your remark should be noted in an other ticket. It's the only mention of `@ManagedBean` in the whole spec. Strange...